src/core/helpers/dates.ts

```diff
@@ -42,3 +42,7 @@ export const dateDiff = (newerDate: Date, olderDate: Date) => {
 };
 
 export const getEndOfDay = (date: Date) => new Date(date.getFullYear(), date.getMonth(), date.getDate(), 23, 59, 59);
+
+export const ascendingSortArray = (array: Array<any>, key: string) => [...array].sort(
```

Comment:
plutot que `any` que penses tu de mettre `object` a minima ? ou meme si c'est possible `{ [key]: Date }` mais je ne sais pas si c'est possible de rendre parametrable le type

Reply:
Tu void comment faire ?
J'ai cherché une solution mais je n'en trouve aucune : 
- le type object ne fonctionne pas
- {[key: string]: Date} non plus car EventType ne rentre pas dans cette catégorie
- Partial<{[key: string]: Date}> pour la même raison
- https://stackoverflow.com/questions/48230773/how-to-create-a-partial-like-that-requires-a-single-property-to-be-set/48244432 -> j'ai testé leur solutions mais sans que rien ne fonctionne

J'ai l'impression qu'il faudrait vraiment creuser ce point pour savoir quoi faire.
J'ai fait une proposition qui contourne le problème.